src/App.tsx

```diff
@@ -133,6 +133,7 @@ const App = () => {
               setState({ ...state, showNav: !state.showNav });
             }}
           />
+          <Banner message={state.user.type}/>
```

Comment:
Would maybe be nice to add a conditional here to hide the banner if state.user.type is null.

Reply:
Oh yeah I do like this idea